### DISCUSSION TITLE
How do I change the sample rate to 44.1kHz?

### DISCUSSION THREAD

**Discussion by tayrod4594** [enzo1982/freac#434] [Help & Support]
I realize this is probably a very simple thing and I'm just too much of a noob to have figured it out, but alas, here we are. I need to convert .wav files to .mp3 files with these specific parameters:

- measure between -23dB and -18dB RMS 
- have -3dB peak values and a max -60dB noise floor
- 192kBps
- CBR 44.1kHz

There are some other requirements as well but I have figured out literally everything but the CBR. How do I set things up in freac to encode at that CBR? I've been tearing my hair out over this and I am desperate for some help. Please and thank you!

**Comment by tayrod4594** [comment]
P.S. I have .freac v.1.1.6, I don't know if that makes a difference.

**Reply by Lee-Carre** [reply]
@tayrod4594

> P.S.

As the OP, you can edit your original post to include more information:
1. Click the horizontal ellipsis (3 horizontal dots: `…`) in the upper-right corner of your OP
2. select `Edit`
3. make constructive changes to the text
4. when finished editing, click the green `Update Post` (or similar, I forget exactly)

> I don't know if that makes a difference.

Citing relevant information, such as which version you're using, is sensible.

**Comment by enzo1982** [comment]
The _CBR_ actually belongs to the 192 kbps line. It means _Constant Bit Rate_, ie. the bitrate will be 192 kbps for the whole file and not adapt to quality requirements. This can be configured in the LAME encoder setttings. Choose _Custom settings_ and then select _CBR_ and a bitrate of 192 kbps:

![grafik](https://user-images.githubusercontent.com/7796713/216735362-7d4b1990-4693-4eca-b6da-423021a26dd5.png)

A sample rate conversion to 44.1 kHz can be configured in the _Processing_ settings in the configuration dialog. On the _Processing_ page, enable the _Sample Rate Converter_ component:

![grafik](https://user-images.githubusercontent.com/7796713/216735683-32c89da4-f4d7-470e-acc8-7945ed50ae68.png)

Then click _Configure component_ and set the _Output sampling rate_ to 44.100 Hz:

![grafik](https://user-images.githubusercontent.com/7796713/216735768-ffb54b5e-f3b3-4ae8-8aa4-9bf2b2dfe32e.png)

You should leave the _Converter_ setting at _Medium Sinc Interpolator_ or set it to _Best Sinc Interpolator_. The latter will be a bit slower, but provide slightly better quality
